test/Benchmarks/Benchmarks.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>$(TargetFrameworksForTests)</TargetFrameworks>
+    <NoWarn>$(NoWarn);SA1518;SA1210</NoWarn>
```

Comment:
@CodeBlanch Please validate if this is a correct way to suppress the proto file message.